Lecture-2/Task2/l2task2/Group.java

```diff
@@ -0,0 +1,29 @@
+package l2task2;
+
+import static java.lang.Math.random;
+
+public class Group {
+
+    private Student[] students;
+    private final int groupId;
+
+    public Group(int groupId, int studentAmount) {
+        this.groupId = groupId;
+        createStudents(studentAmount);
+    }
+
+    public void createStudents(int studentAmount) {
```

Comment:
Creating students is not responsibility of this class. You should pass into `group` already created students array

Reply:
done